include/cuda/std/detail/libcxx/include/cmath

```diff
@@ -318,12 +318,76 @@ long double    truncl(long double x);
 #define NAN      __builtin_nan()
 #endif
 
-_LIBCUDACXX_BEGIN_NAMESPACE_STD
+// Forward to these functions to circumvent macro hackery in some compilers
+namespace {
```

Comment:
Anonymous namespaces are usually a bad idea in headers. I don't understand well enough whats going on here to say if that applies.

Reply:
Removed by getting rid of GCC-5 in our config.